yaml/queueworker-dep.yml

```diff
@@ -21,3 +21,7 @@ spec:
           value: "30s"
         - name: faas_function_suffix
           value: ".openfaas-fn.svc.cluster.local."  # absolute DNS will suppress search domains
+        - name: basic_auth
```

Comment:
What about `yaml_armhf`?

Reply:
is the new version released for `armhf` ?